specification/resources/resource-manager/readme.python.md

```diff
@@ -573,6 +574,16 @@ namespace: azure.mgmt.resource.templatespecs.v2021_05_01
 output-folder: $(python-sdks-folder)/resources/azure-mgmt-resource/azure/mgmt/resource/templatespecs/v2021_05_01
 ```
 
+### Tag: package-templatespecs-2022-02 and python
+
+These settings apply only when `--tag=package-templatespecs-2022-02` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+``` yaml $(tag) == 'package-templatespecs-2022-02'
+namespace: azure.mgmt.resource.templatespecs.v2022_02_01
+output-folder: $(python-sdks-folder)/resources/azure-mgmt-resource/azure/mgmt/resource/templatespecs/v2022_02_01
+```
+
 ``` yaml $(python)
```

Comment:
Should this be `&& $(track2)`?

Reply:
@msyyc to confirm?

Reply:
It was when the PR branch originally diverged, but main no longer includes track2 within this file:

https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/resources/resource-manager/readme.python.md

Reply:
Any update on this?

Reply:
no longer need key word `track2` in `readme.python.md`. Just keep same with main branch